import_dataset/schema_less_guide.md

```diff
@@ -0,0 +1,305 @@
+
+[SVs on Data Commons](https://www.datacommons.org/tools/statvar) are organized and presented in a way that makes them easy to find and understand. They are grouped under different categories such as demographic, economic, and social variables, and are linked to relevant external sources such as the US Census Bureau, Bureau of Labor Statistics, etc.
+
+SVs are comprised of the following elements:
+
+
+
+* <a name="stattype"></a>_[statType](https://datacommons.org/browser/statType)_ refers to a small set of Property nodes
+    * default is `measuredValue`; others include `meanValue`, `minValue`, etc.
+* _[measuredProperty](https://datacommons.org/browser/measuredProperty)_ refers to a Property node
+    * The Property node has _[domainIncludes](https://datacommons.org/browser/domainIncludes)_ of the _[Class](https://datacommons.org/browser/Class)_ referred to in populationType
+* _[populationType](https://datacommons.org/browser/populationType)_ refers to a Class node
+* zero or more constraintProperties with singleton constraint values
+    * These Property nodes have _domainIncludes_ of the Class referred to in populationType
+    * The constraint values are Enumerations or Quantity nodes
+        * Example: `Female`, `[20 Years]`, `AsianAlone`
+* <a name="measurementqualifier"></a>[optional] _[measurementQualifier](https://datacommons.org/browser/measurementQualifier)_ refers to Enumeration nodes
+    * e.g., `Nominal` / `Real` for GDP
+* [optional] _[measurementDenominator](https://datacommons.org/browser/measurementDenominator)_ refers to another StatVar node
+    * For per-capita, `measurementDenominator: dcs:Count_Person`
+
+For a more in-depth look into SVs and an introduction into Statistical Variable Observations (SVOs), see [Representing Statistics in Data Commons](https://github.com/datacommonsorg/data/blob/master/docs/representing_statistics.md).
+
+
+## <a name="schemalesssvs"></a>Schema-less SVs
+
+For datasets with complex schema or ones that we want to import quickly, we can start with a schema-less import, and iteratively add schema. The “schema-less” part of this framework means that the SV is not yet fully defined. This lets us get the dataset into Data Commons without blocking on schema definition.
+
+Schema-less SVs have the following requirements:
+
+
+
+1. The measuredProperty value must match the stat-var DCID.
+* This is important, because this is how we identify it is a schema-less SV. As an example:
+
+    	Node: dcid:Count_MedicalConditionIncident_Tuberculosis
+
+    	typeOf: dcs:StatisticalVariable
+
+    	measuredProperty: dcs:Count_MedicalConditionIncident_Tuberculosis
+
+    	statType: dcs:measuredValue
+
+2. They should have well-formed, stable DCIDs.
+* As we incrementally add schema, we should not change this DCID.
```

Comment:
super nit: please indent this so it falls under bullet (2). here and below

sidenote, in markdown, you can label them all as 1. (instead of numerically increasing them), for easier future updates.